.github/workflows/ci-build-binary-artifacts.yaml

```diff
@@ -154,36 +153,45 @@ jobs:
             -G "${{ matrix.generator }}" ${{ matrix.arch }} \
             -DBUILD_TESTS=OFF \
             -DVCPKG_TRIPLET=${{ matrix.triplet }} \
+            -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} \
             -S .
           cmake --build $BUILD_DIR --parallel --config Release
+          cmake --install $BUILD_DIR
+          cp dependencies.txt ${{ env.INSTALL_DIR }}
 
-      - name: Package
+      - name: Zip artifact
+        shell: bash
+        run: 7z a -tzip pulsar-client-cpp-${{ matrix.triplet }}.zip ${{ env.INSTALL_DIR }}/*
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.triplet }}
+          path: ${{ env.INSTALL_DIR }}
+
+      - name: Build and package (Debug)
         shell: bash
         run: |
-          BUILD_DIR=./build
-          PACKAGE_DIR=./package
-          LIB_DIR=$PACKAGE_DIR/lib/Release
-          VCPKG_INSTALLED_DIR=$PACKAGE_DIR/vcpkg_installed
-          mkdir -p $PACKAGE_DIR
-          mkdir -p $LIB_DIR
-          mkdir -p $VCPKG_INSTALLED_DIR/${{ matrix.triplet }}
-          
-          cp dependencies.txt $PACKAGE_DIR
-          cp -r ./include $PACKAGE_DIR
-          cp -r $BUILD_DIR/include/ $PACKAGE_DIR
-          cp -r $BUILD_DIR/lib/Release/*.lib $LIB_DIR
-          cp -r $BUILD_DIR/lib/Release/*.dll $LIB_DIR
-          cp -r ./vcpkg_installed/${{ matrix.triplet }}/* $VCPKG_INSTALLED_DIR/${{ matrix.triplet }}
+          BUILD_DIR=./build-debug
+          INSTALL_DIR_DEBUG=${{ env.INSTALL_DIR }}-Debug
+          mkdir -p $BUILD_DIR
+          cmake -B $BUILD_DIR \
+            -G "${{ matrix.generator }}" ${{ matrix.arch }} \
+            -DBUILD_TESTS=OFF \
+            -DVCPKG_TRIPLET=${{ matrix.triplet }} \
+            -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR_DEBUG \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -S .
+          cmake --build $BUILD_DIR --parallel --config Debug
+          cmake --install $BUILD_DIR --config Debug
+          cp dependencies.txt $INSTALL_DIR_DEBUG
 
-      - name: Zip artifact
+      - name: Zip artifact (Debug)
         shell: bash
-        run: 7z a -tzip Windows-${{ matrix.triplet }}.zip ./package
+        run: 7z a -tzip pulsar-client-cpp-${{ matrix.triplet }}-Debug.zip ${{ env.INSTALL_DIR }}-Debug/*
 
-      - name: Upload binaries to release
-        uses: svenstaro/upload-release-action@v2
+      - name: Upload artifacts (Debug)
+        uses: actions/upload-artifact@v3
         with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: Windows-${{ matrix.triplet }}.zip
-          asset_name: Windows-${{ matrix.triplet }}.zip
-          tag: ${{ github.ref }}
-          overwrite: true
+          name: ${{ matrix.triplet }}-Debug
```

Comment:
Change `package-linux` job by the way?

Reply:
This PR won't modify the unrelated logic

Reply:
Oh yeah, I got it. I will change it back as well.